graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/ldap/responses/LdapSettingsResponse.java

```diff
@@ -21,7 +21,9 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
+import javax.annotation.Nullable;
 import java.net.URI;
+import java.util.Map;
 
 @JsonAutoDetect
 @AutoValue
```

Comment:
i think the same goes for the ldap config test request and the standard settings dto.
maybe we can clean that up in 2.0?